axelor-account/src/main/resources/reports/AccountingReportType23.rptdesign

```diff
@@ -3110,7 +3110,7 @@ java.lang.String.format(row["translation"],row._outer._outer["dateTo"])
                                     </list-property>
                                     <property name="contentType">html</property>
                                     <text-property name="content"><![CDATA[<VALUE-OF format="HTML">
-if(params["Locale"].value == 'fr') {
+if(params["Locale"].value == 'fr_FR') {
```

Comment:
no you have to extract the language value in the Locale, it has to work even for fr_CA for example